client/my-sites/customer-home/cards/features/blogging-prompt/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { Card, Button, Gridicon } from '@automattic/components';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector, useDispatch } from 'react-redux';
+import CardHeading from 'calypso/components/card-heading';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import useBloggingPrompt from 'calypso/data/blogging-prompt/use-blogging-prompt';
+import useSkipCurrentViewMutation from 'calypso/data/home/use-skip-current-view-mutation';
+import { SECTION_BLOGGING_PROMPT } from 'calypso/my-sites/customer-home/cards/constants';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import BellOffIcon from './bell-off-icon';
+import LightbulbIcon from './lightbulb-icon';
+
+import './style.scss';
+
+export const BloggingPromptCard = () => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSelectedSiteSlug( state ) );
+	const { data: prompt } = useBloggingPrompt( selectedSiteId );
+	const { skipCard } = useSkipCurrentViewMutation( selectedSiteId );
+
+	const hidePrompts = () => {
+		skipCard( SECTION_BLOGGING_PROMPT );
+		dispatch(
+			recordTracksEvent( 'calypso_customer_home_task_skip', {
+				task: SECTION_BLOGGING_PROMPT,
+			} )
+		);
+	};
+	const trackBloggingPromptClick = () => {
+		dispatch(
+			recordTracksEvent( `calypso_customer_home_answer_prompt`, {
+				site_id: selectedSiteId,
+				prompt_id: prompt?.id,
+				prompt_text: prompt?.text,
```

Comment:
Nice to record the prompt id here--will be interesting to see if some prompts are particularly popular!

That said, I don't think including the text in the tracks event is needed--we can always look that up from the prompt id.